### ISSUE TITLE
viewer crashes during save settings

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#431]
I was running on Sierra.  Had saved settings I didn't want to use for a particular session, so ran visit with '-noconfig'.  But I still needed a host profile for sierra to run in parallel, so I used the Options->Host Profiles->Remote profiles mechanisms for importing only the sierra profile.  When I tried to save settings, the viewer segv'd.  I next tried same thing, but removed my .visit directory.  Same segv.  If I don't try to import a sessionfile, but change some other setting then save, then the viewer does not segv.  I have used this mechanism for importing a host profile many times in the past and have never had a problem saving my settings.

tail of viewer.5.vlog:

Xfer::Update: Sending: opcode=47, name=ViewerRPC
Handling WriteConfigFileRPC RPC.
signalhandler_core: SIGSEGV!

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Kathleen Biagas
Original creation: 11/10/2015 01:59 pm
Original update: 11/10/2015 06:30 pm
Ticket number: 2450
